scripts/deploy-website.sh

```diff
@@ -21,10 +21,10 @@ cd $tmp_dir
 git add -A
 git commit \
   --allow-empty \
-  --author "Travis CI <travis-ci@reacttraining.com>" \
+  --author "GitHub CI <github-ci@reacttraining.com>" \
```

Comment:
@chaance **Needs attention:** I've made changes here for consistency, but I'd recommend creating a CI agnostic address like `ci@reacttraining.com` instead.